### PR TITLE
LPS-206653 Orphan social activity data after deleting Virtual Instance

### PR DESCRIPTION
**Pull request by SamZiemer**
https://liferay.atlassian.net/browse/LPS-206653

When deleting social activities, we do not need to check if enabled. Doing so generates an orphaned portalPreferences entry upon deleting a company.

Another option is to check if company deletion is in progress, instead of removing and skipping enabled check if so instead of removing all together.

### CODE REVIEW

**Comment by SamZiemer** [comment]
ci:test:relevant

File: modules/apps/social/social-activity-impl/src/main/java/com/liferay/social/activity/internal/manager/SocialActivityManagerImpl.java [hunk 1]
```diff
@@ -94,10 +94,6 @@ public void addUniqueActivity(
 
 	@Override
 	public void deleteActivities(T model) throws PortalException {
-		if (!_isEnabled(model.getCompanyId())) {
-			return;
-		}
-
```
Comment (marcogalluzzi) [inline modules/apps/social/social-activity-impl/src/main/java/com/liferay/social/activity/internal/manager/SocialActivityManagerImpl.java:100]: Have you tested what happens when `deleteActivities` is triggered for a company having the property `enableUserSocialActivityTracking` disabled? The lack of activities is just ignored or an exception is thrown?

Comment (SamZiemer) [reply modules/apps/social/social-activity-impl/src/main/java/com/liferay/social/activity/internal/manager/SocialActivityManagerImpl.java:100]: Hi Marco, when enableUserSocialActivityTracking is disabled, there are no errors. During the deletion, no records are found and there is simply nothing to delete.

**Review by marcogalluzzi — APPROVED** [review]

**Comment by SylviaLuan** [comment]
ci:test:stable

**Comment by marcogalluzzi** [comment]
ci:test:stable

**Comment by SamZiemer** [comment]
ci:test:stable

**Comment by robertoDiaz** [comment]
ci:report:2528431881

**Comment by robertoDiaz** [comment]
ci:forward

**Pull request closed without merging**